### ISSUE TITLE
Flow Group Schedules Timezones

### ISSUE THREAD

**Issue by zhen0** [PrefectHQ/ui#928]
**Bug Description**

If I create a clock/schedule in the UI, I can set a timezone.  When I create a new clock, the timezone of all clocks is updated. See video for example. 
**Steps To Reproduce**

1. Create a schedule using a cron clock and the timezone selector.  
2. Create a new schedule using the cron clock and timezone selector but select a different timezone. 
3. Note that the clocks initially show different timezone. 
4. Refresh the page - the first clock's timezone is updated. 
5. You can also use the standard clock/schedule creator for the second clock if the UI timezone is different to the timezone of your first clock. On refresh the first clock's schedule will reset to the UI timezone. 

**Browsers Tested:**
* [ ] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Edge
* [ ] IE

----------
**Notes**

The set_flow_group mutation takes a timezone as a main argument so it is set on the flow_group_schedule rather than on an individual clock. 

```mutation {
  set_flow_group_schedule (input: {flow_group_id: "7d71cb62-6852-439b-b7b8-05f7537af09b", timezone: "GMT" }) {
    success
  }
}```

**Comment by zhen0** [comment]
https://user-images.githubusercontent.com/40272060/123708429-668a3080-d839-11eb-87ac-65f55adee617.mov

**Comment by zhen0** [comment]
Closed by #922
